modules/lib/lib-schema/src/main/resources/lib/xp/examples/schema/updatePart.js

```diff
@@ -0,0 +1,85 @@
+var schemaLib = require('/lib/xp/schema');
+var assert = require('/lib/xp/testing');
+
+/* global log*/
+
+let resource = `<?xml version="1.0" encoding="UTF-8"?>
+                <part xmlns="urn:enonic:xp:model:1.0">
+                  <display-name i18n="key.display-name">Virtual Part</display-name>
+                  <description i18n="key.description">My Part Description</description>
+                  <form>
+                    <input type="Double" name="width">
+                      <label i18n="key.label">Column width</label>
+                      <immutable>false</immutable>
+                      <indexed>false</indexed>
+                      <help-text i18n="key.help-text"/>
+                      <occurrences minimum="0" maximum="1"/>
+                    </input>
+                
+                    <mixin name="myapplication:link-urls"/>
+                
+                  </form>
+                  <config>
+                    <input type="Double" name="width">
+                      <label i18n="key.label">Column width</label>
+                      <immutable>false</immutable>
+                      <indexed>false</indexed>
+                      <help-text i18n="key.help-text"/>
+                      <occurrences minimum="0" maximum="1"/>
+                    </input>
+                  </config>
+                </part>
+                `
+
+// BEGIN
+// Update virtual part.
+var result = schemaLib.updateComponent({
+    key: 'myapp:mypart',
+    type: 'PART',
+    resource: resource
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9736743)
```suggestion
    resource
```